cmake/install.cmake

```diff
@@ -87,22 +90,35 @@ if(NOT MSVC)
 else()
   set(CMAKE_INSTALL_CMAKEDIR "cmake" CACHE STRING "${_cmakedir_desc}")
 endif()
+mark_as_advanced(CMAKE_INSTALL_CMAKEDIR)
 
+# Import configuration
 install(EXPORT protobuf-targets
   DESTINATION "${CMAKE_INSTALL_CMAKEDIR}"
   NAMESPACE protobuf::
   COMPONENT protobuf-export)
 
 configure_file(protobuf-config.cmake.in
-  protobuf-config.cmake @ONLY)
+  ${CMAKE_INSTALL_CMAKEDIR}/protobuf-config.cmake @ONLY)
```

Comment:
It is necessary for the subsequent build of the examples as part of the basic process, but examples can also be build separately after the build and installation protobuf (available 2 ways).
Binary folder is similar to the installation folder and has a similar protobuf package configuration files.
Try to enable the option protobuf_BUILD_EXAMPLES.